keg/keg.py

```
@@ -49,16 +36,28 @@ def main():
         Destination directory for generated description, default cwd
 
     -f, --force
-        Force mode (ignore errors, overwrite files)
+       Force mode (ignore errors, overwrite files)
```

is it clear that `ignore errors` will log the warnings ?